theme-data/dark/dark-additional.json

```diff
@@ -150,6 +139,62 @@
       "purple": "@color-purple-70",
       "slate": "@color-slate-70",
       "violet": "@color-violet-70"
+    },
+    "gradientdivider": {
+      "normal": [
+        "@color-white-alpha-00",
+        "@color-white-alpha-20",
```

Comment:
We agreed to only add 4 colours for each of these gradients. Not 2. Because even if they are 4, 2 of them always repeat on each set. It is prob not a big deal if we add 4 of them, but just sth to consider.